generator/routing_index_generator.cpp

```diff
@@ -82,15 +85,18 @@ class VehicleMaskBuilder final
     return mask;
   }
 
-  shared_ptr<IVehicleModel> const m_pedestrianModel;
-  shared_ptr<IVehicleModel> const m_bicycleModel;
-  shared_ptr<IVehicleModel> const m_carModel;
+  shared_ptr<VehicleModelInterface> const m_pedestrianModel;
```

Comment:
Okay, keep VehicleModelInferface name as is, but from my point of view, the suffix "Interface" is redundant. Now it's an interface, but some time later you'll probably make it stateful, with non-pure virtual methods, do you really want to cut the suffix "Interface" then?

Reply:
Я запуталась, у нас уже есть неабстрактные VehicleModel и VehicleModelFactory. Сначала их абстрактные родители VehicleModelInterface и VehicleModelFactoryInterface назывались IVehicleModel и IVehicleModelInterface, но, как ты заметил ранее при ревью, мы не используем префикс I и именуем в соответствии с гугловым стайл гайдом, в котором написано, что для таких случаев можно, но не обязательно использовать суффикс Interface. Если ты предлагаешь назвать абстрактные классы без суффиксов, а для существующих неабстрактных VehicleModel и VehicleModelFactory придумать другое имя, мне кажется от этого будет больше путаницы чем пользы.

Reply:
That's why I'm not insisting on renaming.

The problem is that these classes were incorrectly named initially, and your change generally just expands I to Interface and moves it. I'm not insisting on renaming, as I mentioned above, keep your naming as is. But I suggest you to avoid "Inteface" suffixes for new classes.